pyproject.toml

```diff
@@ -63,14 +64,13 @@ members = ["charms/*"]
 
 [tool.uv.sources]
 ubuntu-drivers-common = { git = "https://github.com/canonical/ubuntu-drivers-common", rev = "554b91edfd3699625dbed90f679abb31a897b76e" }
+hpc-libs = { git = "https://github.com/charmed-hpc/hpc-libs", rev = "1c1fd91bcc7de71fd9da7bc077f88757c81d14ba" }
```

Comment:
[question]: do we need the git hash here because we don't publish tagged revisions for `hpc_libs`?

Out of scope for this PR, but we should discuss the overall structure of `hpc_libs` going forward. We'll likely want to expose exports in the top-level *\_\_init\_\_.py* file, and eventually take `is_container` out and move it to a different module since `systemd-detect-virt` is a relatively generic utility that we can use across multiple projects.

Reply:
> do we need the git hash here because we don't publish tagged revisions for hpc_libs

Yep, pretty much.